.storybook/main.js

```diff
@@ -1,4 +1,7 @@
 module.exports = {
+  features: {
+    previewCsfV3: true,
+  },
```

Comment:
This allows us to use Component Story Format (CSF) version 3 in the stories, and will become the default in Storybook v7.